cmd/kueuectl/app/list/list_clusterqueue.go

```diff
@@ -110,14 +105,19 @@ func (o *ClusterQueueOptions) Complete(clientGetter util.ClientGetter, cmd *cobr
 
 	o.Client = clientset.KueueV1beta1()
 
-	if !o.PrintFlags.OutputFlagSpecified() {
-		o.PrintObj = printClusterQueueTable
-	} else {
+	o.ToPrinter = func(noHeaders bool) (printers.ResourcePrinterFunc, error) {
+		if !o.PrintFlags.OutputFlagSpecified() {
+			printer := newClusterQueueTablePrinter().
+				WithNoHeaders(noHeaders)
```

Comment:
Avoid double negatives
```suggestion
				WithHeaders(headers)
```

Reply:
Fixed